torch_geometric/nn/models/explainer.py

```diff
@@ -114,8 +114,169 @@ def forward(self, mask, *args):
         return x
 
 
-def to_captum(model: torch.nn.Module, mask_type: str = "edge",
-              output_idx: Optional[int] = None) -> torch.nn.Module:
+def _to_edge_mask(edge_index: Tensor) -> Tensor:
+    num_edges = edge_index.shape[1]
+    return torch.ones(num_edges, requires_grad=True, device=edge_index.device)
+
+
+def _raise_on_invalid_mask_type(mask_type: str):
+    if mask_type not in ['node', 'edge', 'node_and_edge']:
+        raise ValueError(f"Invalid mask type (got {mask_type})")
+
+
+def to_captum_input(data: Union[HeteroData, Data],
+                    mask_type: str) -> Tuple[Tuple[Tensor], Tuple[Tensor]]:
```

Comment:
What do you think about adding 
```python
def to_captum_input(data: Union[HeteroData, Data],
                    mask_type: str, *args) -> Tuple[Tuple[Tensor], Tuple[Tensor]]:
```

and add the `*args` directly into the forward_args ? It might be easier to use.

Reply:
Good suggestion. Added it.